mods/wool/init.lua

```diff
@@ -34,19 +34,19 @@ minetest.register_alias("wool:gold", "wool:yellow")
 -- for _,e in ipairs(dye.dyes) do print(("S(%q)"):format(e[2].." Wool")) end
```

Comment:
`-- for _,e in ipairs(dye.dyes) do print(("S(%q)"):format(e[2].." Fabric")) end`

Reply:
Done!